Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM golang:1.16.5-buster AS build-env
+
+# Install minimum necessary dependencies,
+ENV PACKAGES make git gcc
+RUN apt-get update -y
+RUN apt-get install -y $PACKAGES
+
+# Create directory
+RUN mkdir -p /src/panacea-core /src/wasmvm
+
+# Add 'panacea-core' source files
+COPY . /src/panacea-core
+
+# Set working directory for the 'panacea-core' build
+WORKDIR /src/panacea-core
+
+# Install panacea-core
+RUN make clean && make build
+
+# Get 'libwasmvm.so' from wasmvm
+RUN git clone -b v0.14.0 https://github.com/CosmWasm/wasmvm.git /src/wasmvm
```

Comment:
Yeah. There's no other way to get `libwasmvm.so` because `CosmWasm/wasmvm` is indirectly referred by `panacea-core` (The `panacea-core` refers to `CosmWasm/wasmd` which refers to `CosmWasm/wasmvm`). 

Now, we must keep in mind that the version of `wasmvm` must be always updated whenever we update the `wasmd` (in the `go.mod`).

Btw, I just found that the `wasmvm` also provides a `libwasmvm_muslc.a` for Alpine Linux.
https://github.com/CosmWasm/wasmd/blob/d2e35c249e39e90c5a85e03e0b75800431612b20/Dockerfile#L18
It would be better to use that, but not for now because that's a static library (not a shared library). It means that we have to change our Makefile to build the `panacead` with the static library. So, let's do it later. For now, your strategy is good enough.

Reply:
OK!